cross/ariang/PLIST

```diff
@@ -0,0 +1,24 @@
+rsc:web/LICENSE
+rsc:web/css/aria-ng-4a38f3a4ab.min.css
+rsc:web/css/bootstrap-3.4.1.min.css
+rsc:web/css/plugins-ccac6fc3fc.min.css
+rsc:web/favicon.ico
+rsc:web/favicon.png
+rsc:web/fonts/fontawesome-webfont.eot
+rsc:web/fonts/fontawesome-webfont.svg
+rsc:web/fonts/fontawesome-webfont.ttf
+rsc:web/fonts/fontawesome-webfont.woff
+rsc:web/fonts/fontawesome-webfont.woff2
+rsc:web/index.html
+rsc:web/js/angular-packages-1.6.10.min.js
+rsc:web/js/aria-ng-a0b72e89e6.min.js
+rsc:web/js/bootstrap-3.4.1.min.js
+rsc:web/js/echarts-common-3.8.5.min.js
+rsc:web/js/jquery-3.3.1.min.js
+rsc:web/js/moment-with-locales-2.29.4.min.js
+rsc:web/js/plugins-b3cb190423.min.js
+rsc:web/langs/zh_Hans.txt
+rsc:web/langs/zh_Hant.txt
+rsc:web/robots.txt
+rsc:web/tileicon.png
+rsc:web/touchicon.png
```

Comment:
you can simplify this (makes it independent of most changes):
```suggestion
rsc:web/
```

Reply:
Thank you, I didn't knew you can put there a whole directory. It will simplify the PLIST, but there will be many `*.md` files with documentation included in the final package.

I however agree that the PLIST would have to be changed w.r.t. version strings in the javascripts and csss (csses? css's?) so maybe it would be easier to maintain with just a `rsc:web/`.

Reply:
@vitkabele I did it (together with an update to v1.3.7)